gammapy/data/observations.py

```diff
@@ -346,51 +346,56 @@ def check(self, checks="all"):
         checker = ObservationChecker(self)
         return checker.run(checks=checks)
 
-    def peek(self, figsize=(12, 10)):
+    def peek(self, figsize=(15, 10)):
         """Quick-look plots in a few panels.
 
         Parameters
         ----------
         figsize : tuple
             Figure size
         """
-        n_irfs = len(self.available_hdus)
+        plot_hdus = self.available_hdus
```

Comment:
It might be useful to take the intersection of `available_hdus` with a set of plottable hdus. This avoids removing explicitly `gti` and would ensure that no additional sub_plot is created when another hdu is present.